README.md

```diff
@@ -62,6 +62,37 @@ _For both Windows and Mac : To verify that installation was successful, `protoc
 
 ### Config Format
 
+#### Exclude Paths: ####
+
+This allows the user to blacklist a certain set of paths for a given path. As a result those paths would be eliminated while grabbing content, there are certain rules that need to be followed,
+
+1. Exclude Paths only accepts relative paths and not absolute paths.
+
+    For example,
+
+    Let's say given path : `"/content/someContent"`  and you would like to remove `"/content/someContent/someOtherContent/pdfs"`
```

Comment:
yup caught that, was waiting for any more changes for the readme file before I push the final commit.